### PR TITLE
Styling teams

### PR DESCRIPTION
**Pull request by Yehezkiel01**
I make from `teams` endpoint redirect to `teams/2020`, to make it easier.

### CODE REVIEW

File: src/routes/team/_constants.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/* This file contains constants value specific to the team pages */
+export const current_year = 2020;
```
Comment (benclmnt) [inline src/routes/team/_constants.js]: Pls add a note in our README.md to change this every semester.

File: src/routes/team/index.svelte [hunk 2]
```diff
@@ -1,37 +1,7 @@
 <script context="module">
+    import * as constants from './_constants.js';
```
Comment (benclmnt) [inline src/routes/team/index.svelte]:
```suggestion
    import { current_year } from './_constants.js';
```

File: src/routes/team/index.svelte [hunk 3]
```diff
@@ -1,37 +1,7 @@
...
     export function preload({ params, query }) {
-        return this.fetch(`team.json`)
-            .then((r) => r.json())
-            .then((res) => {
-                return {
-                    team: res.team,
-                    year: res.year,
-                };
-            });
+        return this.redirect(302, `team/` + constants.current_year);
```
Comment (benclmnt) [inline src/routes/team/index.svelte]:
```suggestion
        return this.redirect(302, `team/` + current_year);
```

File: src/routes/team/[slug].svelte [hunk 4]
```diff
@@ -18,31 +18,102 @@
...
+            Previous year
+        </a>
+    {/if}
+
+    {#if Number(year) < constants.current_year}
+        <a class="navigation-button right" href="team/{Number(year) + 1}">
+            Next year
+        </a>
+    {/if}
+</div>
```
Comment (benclmnt) [inline src/routes/team/[slug].svelte]:
```suggestion
<div class="navigation">
    {#if Number(year) > first_year}
        <a class="navigation-button left" href="team/{Number(year) - 1}">
            Previous year
        </a>
    {/if}

    {#if Number(year) < current_year}
        <a class="navigation-button right" href="team/{Number(year) + 1}">
            Next year
        </a>
    {/if}
</div>
```

File: src/routes/team/[slug].svelte [hunk 5]
```diff
@@ -18,31 +18,102 @@
 </script>
 
 <script>
+    import * as constants from './_constants.js';
```
Comment (benclmnt) [inline src/routes/team/[slug].svelte]:
```suggestion
    import { current_year, first_year } from './_constants.js';
```

**Review by benclmnt — CHANGES_REQUESTED** [review]
I will wait for the photos before approving. 

Can you decrease the space between name and job title?

File: src/routes/team/_constants.js (hunk 1 above)
Comment (Yehezkiel01) [reply src/routes/team/_constants.js]: Sure

File: src/routes/team/index.svelte (hunk 2 above)
Comment (Yehezkiel01) [reply src/routes/team/index.svelte]: Thanks, this is way simpler 👍

**Review by benclmnt — APPROVED** [review]
ok!!

**Pull request merged**